### PR TITLE
AR-15848 - Fixup documentation

### PR DESCRIPTION
**Pull request by VinnieMaida**
@NextechSystems/the-architects 

This should take care of...a lot. This is a mixture of trying to make all of our R4 documentation consistent and also just fixing up things that were either missing, wrong, or incomplete. Most, but not all, of the things I found and fixed up here are described in this comment: https://nextech.atlassian.net/browse/AR-16061?focusedCommentId=969643. I tried to split out all of the commits into different resources/fixes, for the most part, to try and make everything easier to follow.

### CODE REVIEW

File: source/includes/r4/_account.md [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
-| type | Categorizes the account for reporting and searching purposes. | [CodeableConcept](https://www.hl7.org/fhir/datatypes.html#CodeableConcept) | _12.9_ |
-| balance | Represents the sum of all credits less all debits associated with the account. Might be positive, zero or negative. | [Money](https://www.hl7.org/fhir/datatypes.html#Money) | _12.9_ |
-| description | Provides additional information about what the account tracks and how it is used. | [string](https://www.hl7.org/fhir/datatypes.html#String) | _12.9_ |
-| subject | Identifies the patient or other object the account is associated with. | [Reference](https://www.hl7.org/fhir/references.html) | _12.9_ |
-| guarantor | Party financially responsible for the account. | [BackboneElement](https://www.hl7.org/fhir/backboneelement.html) | _12.9_ |
-| guarantor.party | The entity who is responsible. | [Reference](https://www.hl7.org/fhir/references.html) | _12.9_ |
+| id | Unique identifier used to reference the account. | [string](https://www.hl7.org/fhir/R4/datatypes.html#string) | _12.9_ |
```
Comment (JakeDriscoll) [inline source/includes/r4/_account.md:9]: is `12.9` the correct version for this? I'm assuming so just making sure R4 works that way for my own education.

File: source/includes/r4/_account.md [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
+| identifier | Unique identifier used to reference the account. | [Identifier](https://www.hl7.org/fhir/R4/datatypes.html#Identifier) | _12.9_ |
+| name |The name associated with the Account. | [string](https://www.hl7.org/fhir/R4/datatypes.html#string) | _12.9_ |
+| status | Indicates whether the account is presently used/usable or not. | [CodeableConcept](https://www.hl7.org/fhir/R4/datatypes.html#CodeableConcept) | _12.9_ |
+| type | Categorizes the account for reporting and searching purposes. | [CodeableConcept](https://www.hl7.org/fhir/R4/datatypes.html#CodeableConcept) | _12.9_ |
+| balance | Represents the sum of all credits less all debits associated with the account. Might be positive, zero or negative. | [Money](https://www.hl7.org/fhir/R4/datatypes.html#Money) | _12.9_ |
```
[the remark this answers is not in the document]
Comment (VinnieMaida) [reply source/includes/r4/_account.md:14]: Good call. I suspect it was just copied over from the STU3 docs but, looking at the R4 mapper, it looks like we are indeed not returning `balance` for R4, so I'll take it out. Thanks!

File: source/includes/r4/_organization.md [hunk 3]
```diff
@@ -1,20 +1,19 @@
...
-| telecom | The contact details of communication at the location | [ContactPoint](https://www.hl7.org/fhir/datatypes.html#ContactPoint) | _12.8_ |
-| address | The address of the location | [Address](https://www.hl7.org/fhir/datatypes.html#Address) | _12.8_ |
-| address.country | The country of the location address | [string](https://www.hl7.org/fhir/datatypes.html#string) | _16.9_ |
+| id | The unique value assigned to each location which discerns it from all others | [string](https://www.hl7.org/fhir/R4/datatypes.html#string) | _12.8_ |
+| identifier | The unique value assigned to each location which discerns it from all others | [Identifier](https://www.hl7.org/fhir/R4/datatypes.html#Identifier) | _12.8_ |
+| meta.lastUpdated | The last time the organization was modified | [instant](https://www.hl7.org/fhir/R4/datatypes.html#instant) | _16.9_ |
+| identifier | NPI identifier for the organization that is used to identify the organization across multiple disparate systems |  [Identifier](https://www.hl7.org/fhir/us/core/STU3.1.1/StructureDefinition-us-core-organization-definitions.html#Organization.identifier:NPI) | _16.9_ |
```
[the remark this answers is not in the document]
Comment (VinnieMaida) [reply source/includes/r4/_organization.md:12]: Good call! I didn't notice that we're returning both the regular ID and the NPI in the `identifier` list for this particular resource.

File: source/includes/r4/_patient.md [hunk 4]
```diff
@@ -2032,136 +1952,152 @@ GET https://select.nextech-api.com/api/r4/Encounter?_id=123
...
-| lastUpdated | The date a goal was last modified | [dateTime](https://build.fhir.org/datatypes.html#dateTime) | _12.6_ |
+| identifier | The unique value assigned to each goal which discerns them from all others. | [Identifier](https://www.hl7.org/fhir/R4/datatypes.html#Identifier) | _17.0_ |
+| id | The unique value assigned to each goal which discerns them from all others. | [string](https://www.hl7.org/fhir/R4/datatypes.html#string) | _17.0_ |
+| subject | The patient pertaining to the goal | [Reference](https://www.hl7.org/fhir/R4/references.html)([US Core Patient Profile](https://hl7.org/fhir/us/core/STU3.1.1/StructureDefinition-us-core-patient.html)) | _17.0_ |
+| target | The target outcome of the goal | [CodeableConcept](http://hl7.org/fhir/R4/datatypes.html#CodeableConcept) | _17.0_ |
+| note | Comments about the goal | [Annotation](https://www.hl7.org/fhir/R4/datatypes.html#Annotation) | _17.0_ |
+| date | The date or date range of the goal or goals | [dateTime](https://www.hl7.org/fhir/R4/datatypes.html#dateTime) | _17.0_ |
+| meta.lastUpdated | The last time the goal was modified | [instant](https://hl7.org/fhir/R4/datatypes.html#instant) | _17.0_ |
```
[the remark this answers is not in the document]
Comment (VinnieMaida) [reply source/includes/r4/_patient.md:1973]: Good call! Looks like those indeed were never added. I'll throw them in!

**Review by joerocca — APPROVED** [review]
Looks good!

**Pull request merged**